core/templates/domain/exploration/ExplorationObjectFactory.ts

```diff
@@ -176,7 +176,11 @@ export class Exploration {
     return this.getState(stateName).content.html;
   }
 
-  getAllVoiceovers(languageCode: string): VoiceoverObjectsDict {
+  // If there is no languagecode provided null will returned.
+  getAllVoiceovers(languageCode: string | null): VoiceoverObjectsDict | null {
```

Comment:
Can we make sure that we don't call this function with `null`?

Reply:
Yes, done.